docs/CONTRIBUTING.md

```diff
@@ -111,6 +114,24 @@ The usage of host.docker.internal comes from the [Docker networking documentatio
 
 There is currently no arm64 support for the go1.x runtime, see [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). Known issues running on linux/arm64 include the inability to network with the localhost from the public.ecr.aws/lambda/go Docker image.
 
+### Testing in Docker
+
+We support local testing in Docker. Ensure ``docker`` is installed and running, and then run:
```

Comment:
Should we include something about setting the DRIVER_TOOLS environment variable, rather than being informed only via error? E.g.

>    To test using `docker`, you will need to set the `DRIVERS_TOOLS` environment variable to point to a local clone of the [drivers-evergreen-tools](https://github.com/mongodb-labs/drivers-evergreen-tools) repository. This is essential for running the testing matrix in a container. You can set the `DRIVERS_TOOLS` variable in your shell profile or in your project-specific environment.

Reply:
I applied this comment in the follow up PR: #1430